cpp/oneapi/dal/data/undirected_adjacency_array_graph.hpp

```diff
@@ -90,19 +90,20 @@ class ONEAPI_DAL_EXPORT undirected_adjacency_array_graph {
     using vertex_edge_range       = range<vertex_edge_iterator>;
     using const_vertex_edge_range = range<const_vertex_edge_iterator>;
 
+    static_assert(std::is_integral_v<vertex_type> && std::is_signed_v<vertex_type> && sizeof(vertex_type) == 4, "Wrong vertex type. It must be 4 bytes signed integer");
+
     undirected_adjacency_array_graph();
-    undirected_adjacency_array_graph(undirected_adjacency_array_graph&& graph)      = default;
-    undirected_adjacency_array_graph(const undirected_adjacency_array_graph& graph) = default;
+    undirected_adjacency_array_graph(undirected_adjacency_array_graph &&graph);
```

Comment:
Align the style - & or && should be moved to the left like here: type& (not type &)
The same about const ref. Const should be on the left side from the type: const type& (not type const &)